src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -29,32 +30,36 @@ export type ActionName =
     | "Download Driver Overview"
     | "Delete Parcel";
 
-type AvailableActionsType = {
-    [actionKey in ActionName]: {
-        newStatus: StatusType;
-    };
+type ActionTypes = {
+    actionName: ActionName;
+    newStatus: StatusType;
+    availableToVolunteers: boolean;
 };
 
-export const availableActions: AvailableActionsType = {
-    "Download Day Overview": {
+const availableActions: ActionTypes[] = [
+    {
+        actionName: "Download Day Overview",
         newStatus: "Day Overview Downloaded",
+        availableToVolunteers: true,
     },
-    "Download Shopping Lists": {
+    {
+        actionName: "Download Shopping Lists",
         newStatus: "Shopping List Downloaded",
+        availableToVolunteers: true,
     },
-    "Download Shipping Labels": {
+    {
+        actionName: "Download Shipping Labels",
         newStatus: "Shipping Labels Downloaded",
+        availableToVolunteers: true,
     },
-    "Generate Map": {
-        newStatus: "Map Generated",
-    },
-    "Download Driver Overview": {
+    { actionName: "Generate Map", newStatus: "Map Generated", availableToVolunteers: true },
```

Comment:
Is Eslint forcing it on one line? Would the linter complain if you put that object on three lines like the other ones just so it looks nicer and consistent. Annoying if it does.

Reply:
Working now 👍